bitnami/mysql/README.md

```diff
@@ -46,222 +46,274 @@ The command removes all the Kubernetes components associated with the chart and
-| `metrics.serviceMonitor.honorLabels`      | honorLabels chooses the metric's labels on collisions with target labels.           | `false`                                                      |
-| `metrics.serviceMonitor.additionalLabels` | Used to pass Labels that are required by the Installed Prometheus Operator          | `{}`                                                         |
-| `metrics.serviceMonitor.release`          | Used to pass Labels release that sometimes should be custom for Prometheus Operator | `nil`                                                        |
+| Name                                         | Description                                                                                                           | Value                     |
+| -------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- | ------------------------- |
+| `metrics.enabled`                            | Start a side-car prometheus exporter                                                                                  | `false`                   |
+| `metrics.image.registry`                     | Exporter image registry                                                                                               | `docker.io`               |
+| `metrics.image.repository`                   | Exporter image repository                                                                                             | `bitnami/mysqld-exporter` |
+| `metrics.image.tag`                          | Exporter image tag (immutable tags are recommended)                                                                   | `0.13.0-debian-10-r19`    |
+| `metrics.image.pullPolicy`                   | Exporter image pull policy                                                                                            | `IfNotPresent`            |
+| `metrics.image.pullSecrets`                  | Specify docker-registry secret names as an array                                                                      | `[]`                      |
+| `metrics.service.type`                       | Kubernetes service type for MySQL Prometheus Exporter                                                                 | `ClusterIP`               |
+| `metrics.service.port`                       | MySQL Prometheus Exporter service port                                                                                | `9104`                    |
+| `metrics.service.annotations`                | Prometheus exporter service annotations                                                                               | `{}`                      |
+| `metrics.extraArgs`                          | Extra args to be passed to mysqld_exporter                                                                            | `{}`                      |
```

Comment:
I can see in the values:

```
  extraArgs:
    primary: []
    secondary: []
```
I think primary and secondary are missing.

Reply:
Yes, in this case I opted for documenting just the parent object but I see they are in the original README so I've updated it in d152d52647a60f9eb9b85b13c46f8e6d515ffdf9. Thanks!